modules/go-add-infra-nodes.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="add-infra-nodes_{context}"]
+= Adding Infra Nodes
+
+.Procedure
+
+. Label the existing nodes as Infrastructure usinf the following command:app-name:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra
+----
+. Add `runOnInfra` toggle in the `GitOpsService` Custom Resource (CR) to add infra node selector:
++
+[source,terminal]
+----
+$ oc edit gitopsservice -n openshift-gitops
+----
+. Add `runOnInfra` to spec and set to `true` in the YAML:
```

Comment:
```suggestion
. In the ????.yml file, add `runOnInfra` to the `spec` section and set to it to `true`:
```
- It's not clear to me that the first part of this step is telling the user to add a section to the YAML file. Maybe phrase it per my suggestion to make this step more clear.
- Also, indicate what YML file the user should be altering as part of this process.